cmd/rcat/rcat.go

```diff
@@ -37,6 +44,9 @@ must fit into RAM. The cutoff needs to be small enough to adhere
 the limits of your remote, please see there. Generally speaking,
 setting this cutoff too high will decrease your performance.
 
+Use the |--size| flag to preallocate the file in advance at the remote end
+and actually stream it, even if remote backend doesn't support streaming.
+
```

Comment:
I think this needs a bit more explanation - how about this as well?

```
|--size| should be the exact size of the input stream in bytes. If the
size of the stream is different in length to the |--size| passed in
then the transfer will likely fail.
```

Reply:
Yep, I agree. I was actually testing this locally with `zfs send` and I had to use `split --filter` and `head -c /dev/zero` to pad the stream to correct length.